workshops/views.py

```diff
@@ -1771,6 +1771,7 @@ def workshop_staff(request):
                 people = people.filter(q, task__role__name='learner') \
                                .exclude(badges__in=instructor_badges)
 
+    emails = people.values_list('email', flat=True)
```

Comment:
Note: The list of emails here is the entire list of staff that confirm to the filter given. They may be listed in the second or the third page when the results are paginated.

@gvwilson @maneesha Do we want to email the list shown on the particular page number or do we want to email every staff in all the page numbers.